.drone.yml

```diff
@@ -0,0 +1,93 @@
+kind: pipeline
+name: default
+
+volumes:
+- name: cache
+  temp: {}
+
+steps:
+- name: build
+  image: plugins/docker
+  settings:
+    tag: latest
+    repo: docker.netsoc.co/public/netsoc.co_new
```

Comment:
do we wanna just overwrite the whole website? :man_shrugging:

Reply:
Eventually, yes I would imagine, but for the time being, keep them separate until full release of new website, and maybe archive the future "old" website?

Reply:
I mean it's an easy change when the websites working